lib/texting.rb

```diff
@@ -0,0 +1,34 @@
+# This class handles the sms functionality with twilio
+require 'twilio-ruby'
+require 'customer'
+
+
+class SmsText
+
+  # sends the message
+  def send_text
+    init_msg(confirm_order)
+  end
+
+  private
+
+  # Links functionality with twilio
+  def init_msg(msg)
+    account_sid = 'ACcb0e21414d3ab3d87192fa33e93fc1c0' # => please change if testing
+    auth_token = 'fc23137ee0ee4262f9c677df35211a4d'    # => please change if testing
+  # set up a client to talk to the twilio REST API
```

Comment:
Please remove these credentials from GitHub - look into using environment variables to protect this data by storing it locally only.